packages/data/published-types/index.d.ts

```diff
@@ -32,7 +34,8 @@ export function attr(): PropertyDecorator;
  * @function
  * @param {Object} [options] - Optional attribute options
  */
-export function attr(options?: DS.AttrOptions<any>): PropertyDecorator;
+// tslint:disable-next-line:unified-signatures
+export function attr(options?: DS.AttrOptions): PropertyDecorator;
```

Comment:
`DS.AttrOptions` already has a default generic type argument of `any`. There's no need to explicitly specify one